android/source/VideoLocker/res/layout/activity_course_video_list.xml

```diff
@@ -7,7 +7,7 @@
 
     <fragment
         android:id="@+id/fragment_player"
-        android:name="org.edx.mobile.view.VideoListFragment"
+        android:name="org.edx.mobile.player.VideoListFragment"
```

Comment:
Looks like <code>activity_course_video_list.xml</code> is unsed file. Please check and delete if so.